packages/react-ui-kit/src/Form/IndicatorRangeInput.styles.ts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Wire
+ * Copyright (C) 2022 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {CSSObject} from '@emotion/react/dist/emotion-react.cjs';
+
+import {COLOR_V2} from '../Identity';
+import {manySelectors} from '../util';
+
+const thumbSelectors = ['&::-webkit-slider-thumb', '&::-moz-range-thumb', '&::-ms-thumb'];
+const sliderSelectors = ['&::-webkit-slider-runnable-track', '&::-moz-range-track', '&::-ms-track'];
+
+export const rangeStyles = (
+  backgroundSize: string,
+  value: number,
+  listLength: number,
+  isCustomSlider: boolean,
+): CSSObject => {
+  const firstOptionThumbPosition = `calc(((100% - 10px) / (${listLength} * 2) - 4px) / 2)`;
+  const lastOptionThumbPosition = `calc(((100% - 12px) / (${listLength} * 2) - 6px) / 2)`;
+
+  const isFirstOption = value === 0;
+  const isLastOption = value === listLength;
+
+  return {
+    appearance: 'none',
+    width: '100%',
+    height: '8px',
+    backgroundImage: `linear-gradient(${COLOR_V2.BLUE_LIGHT_700}, ${COLOR_V2.BLUE_LIGHT_700})`,
+    backgroundColor: COLOR_V2.GRAY_60,
+    backgroundSize: backgroundSize || '0% 100%',
+    backgroundRepeat: 'no-repeat',
+    display: 'flex',
+    alignItems: 'center',
+    borderRadius: '8px',
+    marginBlock: '10px',
+    zIndex: '1',
+    position: 'relative',
+
+    ...manySelectors(thumbSelectors, {
+      '-webkit-appearance': 'none',
```

Comment:
It's usually considered good practice to specify the vendor-prefixed version first and then the non-prefixed version, in order that the non-prefixed property will override the vendor-prefixed property-settings once it's implemented;

according to that, appearance should come after '-webkit-appearance'. In this case it doesn't matter much because both are having the same value but in future we may change it.